scripts/build/build_node/Platform/Linux/install-ubuntu-build-tools.sh

```diff
@@ -23,21 +23,25 @@ then
     exit 1
 fi
 
-UBUNTU_DISTRO="$(lsb_release -c | awk '{print $2}')"
-if [ "$UBUNTU_DISTRO" == "bionic" ]
-then
-    echo "Setup for Ubuntu 18.04 LTS ($UBUNTU_DISTRO)"
-elif [ "$UBUNTU_DISTRO" == "focal" ]
-then
-    echo "Setup for Ubuntu 20.04 LTS ($UBUNTU_DISTRO)"
-elif [ "$UBUNTU_DISTRO" == "jammy" ]
+#
+# Get Ubuntu version data
+#
+UBUNTU_DISTRO="$(lsb_release -sc)"
+UBUNTU_VER="$(lsb_release -sr)"
+if [ "$UBUNTU_DISTRO" == "bionic" ] || [ "$UBUNTU_DISTRO" == "focal" ] || [ "$UBUNTU_DISTRO" == "jammy" ]
```

Comment:
Note that ROS2-Gem supports only Humble and Galactic ROS2. ROS2 Galactic is EOL, so only Ubuntu 22.04 and Humble should be supported for ROS2-Gem.

Reply:
Good call out, I'll remove the package from the 20.04 (focal) package list

Reply:
Actually on second thought, I'll leave it in for our CI processes, but we'll call out that it's not supported in docs